pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -58,10 +58,6 @@ const (
 	// desiredStateOfWorldPopulatorLoopSleepPeriod is the amount of time the
 	// DesiredStateOfWorldPopulator loop waits between successive executions
 	desiredStateOfWorldPopulatorLoopSleepPeriod time.Duration = 1 * time.Minute
-
-	// reconcilerSyncDuration is the amount of time the reconciler sync states loop
-	// wait between successive executions
-	reconcilerSyncDuration time.Duration = 5 * time.Second
```

Comment:
Can you summarize what the impact of lengthening this sync interval is for volumes in general? The referenced issue talks about the downside  of synchronizing AWS volumes frequently, but I assume there's also a downside of synchronizing less frequently (or never). Does that impact pod startup time? Termination time? Correction of volume mount issues? Need to document that somewhere if this is tunable and we want cluster admins to understand the trade offs. It also seems like tuning a single sync period for all volumes types will be difficult if some (like AWS) are relatively expensive and some (like hostPath or NFS) or relatively cheap